Android/app/src/main/java/com/example/androidgui/inventoryactivities/UpdateQuanActivity.java

```diff
@@ -24,16 +28,22 @@ protected void onCreate(Bundle savedInstanceState) {
         name = findViewById(R.id.editTextTextPersonName6);
         usage = findViewById(R.id.editTextTextPersonName7);
         btn.setOnClickListener(this);
+        ip.setUpdateQuanInterface(this);
     }
     @Override
     public void onClick(View v) {
         String iname = name.getText().toString();
         String iusage = usage.getText().toString();
         String message = im.newQuantity(iname,iusage,UpdateQuanActivity.this);
-//        im.SavetoFile();
-        Toast.makeText(UpdateQuanActivity.this,message,Toast.LENGTH_SHORT).show();
+        ip.showInfo(message);
     }
     public String getMessage(String message){
```

Comment:
public methods in activities shouldn't be returning things. This should probably be private